<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>cEdit</class>
 <widget class="QDialog" name="cEdit">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>804</width>
    <height>732</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Edit</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="0" column="0">
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <widget class="QTabWidget" name="m_lpTabWidget">
       <property name="currentIndex">
        <number>0</number>
       </property>
       <widget class="QWidget" name="tab">
        <attribute name="title">
         <string>Progress</string>
        </attribute>
        <layout class="QGridLayout" name="gridLayout">
         <item row="0" column="0">
          <layout class="QVBoxLayout" name="verticalLayout">
           <item>
            <layout class="QVBoxLayout" name="m_lpVerticalLayout">
             <item>
              <layout class="QHBoxLayout" name="horizontalLayout_6">
               <item>
                <widget class="QLineEdit" name="m_lpName">
                 <property name="readOnly">
                  <bool>true</bool>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLabel" name="label_4">
                 <property name="text">
                  <string>Year:</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QDateEdit" name="m_lpFirstAired"/>
               </item>
               <item>
                <widget class="QCheckBox" name="m_lpCliffhanger">
                 <property name="text">
                  <string>Cliffhanger</string>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
             <item>
              <layout class="QHBoxLayout" name="horizontalLayout_2">
               <item>
                <widget class="QLabel" name="label">
                 <property name="text">
                  <string>Download Link:</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLineEdit" name="m_lpDownloadLink"/>
               </item>
              </layout>
             </item>
             <item>
              <layout class="QHBoxLayout" name="horizontalLayout">
               <item>
                <widget class="QPushButton" name="m_lpAllInit">
                 <property name="text">
                  <string>all Init</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QPushButton" name="m_lpAllProgress">
                 <property name="text">
                  <string>all Progress</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QPushButton" name="m_lpAllDone">
                 <property name="text">
                  <string>all Done</string>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
             <item>
              <widget class="QScrollArea" name="m_lpScrollArea">
               <property name="widgetResizable">
                <bool>true</bool>
               </property>
               <widget class="QWidget" name="m_lpScrollAreaWidget">
                <property name="geometry">
                 <rect>
                  <x>0</x>
                  <y>0</y>
                  <width>754</width>
                  <height>546</height>
                 </rect>
                </property>
                <layout class="QGridLayout" name="m_lpGridLayoutScroll"/>
               </widget>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="tab_2">
        <attribute name="title">
         <string>Details</string>
        </attribute>
        <layout class="QGridLayout" name="gridLayout_3">
         <item row="0" column="0">
          <layout class="QVBoxLayout" name="verticalLayout_3" stretch="0,1,1">
           <item>
            <widget class="QLabel" name="m_lpDetailsBanner">
             <property name="minimumSize">
              <size>
               <width>758</width>
               <height>140</height>
              </size>
             </property>
             <property name="frameShape">
              <enum>QFrame::Box</enum>
             </property>
             <property name="text">
              <string/>
             </property>
             <property name="alignment">
              <set>Qt::AlignCenter</set>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QScrollArea" name="scrollArea">
             <property name="widgetResizable">
              <bool>true</bool>
             </property>
             <widget class="QWidget" name="scrollAreaWidgetContents">
              <property name="geometry">
               <rect>
                <x>0</x>
                <y>0</y>
                <width>756</width>
                <height>241</height>
               </rect>
              </property>
              <layout class="QGridLayout" name="gridLayout_4">
               <item row="0" column="0">
                <layout class="QHBoxLayout" name="horizontalLayout_5">
                 <item>
                  <widget class="QTextEdit" name="m_lpDetailsOverview">
                   <property name="readOnly">
                    <bool>true</bool>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <layout class="QVBoxLayout" name="verticalLayout_4" stretch="2,1">
                   <item>
                    <layout class="QHBoxLayout" name="horizontalLayout_4">
                     <item>
                      <widget class="cVerticalLabel" name="label_2">
                       <property name="text">
                        <string>Actors</string>
                       </property>
                      </widget>
                     </item>
                     <item>
                      <widget class="QTreeWidget" name="m_lpDetailsActors">
                       <attribute name="headerVisible">
                        <bool>false</bool>
                       </attribute>
                       <column>
                        <property name="text">
                         <string notr="true">1</string>
                        </property>
                       </column>
                      </widget>
                     </item>
                    </layout>
                   </item>
                   <item>
                    <layout class="QHBoxLayout" name="horizontalLayout_3">
                     <item>
                      <widget class="cVerticalLabel" name="label_3">
                       <property name="text">
                        <string>Genre</string>
                       </property>
                      </widget>
                     </item>
                     <item>
                      <widget class="QTreeWidget" name="m_lpDetailsGenre">
                       <attribute name="headerVisible">
                        <bool>false</bool>
                       </attribute>
                       <column>
                        <property name="text">
                         <string notr="true">1</string>
                        </property>
                       </column>
                      </widget>
                     </item>
                    </layout>
                   </item>
                  </layout>
                 </item>
                </layout>
               </item>
              </layout>
             </widget>
            </widget>
           </item>
           <item>
            <widget class="QTabWidget" name="m_lpDetailsSeasonTab">
             <property name="currentIndex">
              <number>0</number>
             </property>
             <widget class="QWidget" name="tab_3">
              <attribute name="title">
               <string>Tab 1</string>
              </attribute>
             </widget>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </widget>
     </item>
     <item>
      <widget class="QDialogButtonBox" name="m_lpButtonBox">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="standardButtons">
        <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>cVerticalLabel</class>
   <extends>QLabel</extends>
   <header>cverticallabel.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>m_lpButtonBox</sender>
   <signal>accepted()</signal>
   <receiver>cEdit</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>m_lpButtonBox</sender>
   <signal>rejected()</signal>
   <receiver>cEdit</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
